client/post-editor/editor-feedback-invitation/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+function EditorFeedbackInvitation( { translate, onTrigger } ) {
+	return (
+		<div className="editor-feedback-invitation">
+			<p className="editor-feedback-invitation__invitation">
+				{ translate(
+					'{{highlight}}Need another set of eyes?{{/highlight}} Ask a friend to review your post.',
+					{
+						components: {
+							highlight: <span className="editor-feedback-invitation__highlight" />
+						}
+					}
+				) }
+			</p>
+			<Button className="editor-feedback-invitation__button" onClick={ onTrigger }>
+				{ translate( 'Send to a Friend' ) }
+			</Button>
+		</div>
+	);
+}
```

Comment:
Fun fact! (and purely personal style) we can remove some syntax noise here with the arrow function syntax. Further, we can export the component itself _unwrapped_ so as to make testing it in isolation easier.

```js
export const EditorFeedbackInvitation = ( { onTrigger, translate } ) => (
	<div …>
		…
	</div>
);

export default localize( EditorFeedbackInvitation );

// other.jsx
import EditorFeedbackInvitation from …

<EditorFeedbackInvitation onTrigger={ grillABurger } />

// test.jsx
import 

const invite = shallow( <EditorFeedbackInvitation translate={ identity } onTrigger={ triggerSpy } /> );
expect( invite.blah.blah ).to.equal( 'Send to a Friend' );
```